Gek62/hometask-5/5.4.8.sql

```diff
@@ -0,0 +1,3 @@
+DELETE 
+FROM film CASCADE
```

Comment:
Does it even work?
I think, no. Try a different year, which exists in the database (2013 does not), and you'll see an error caused by the foreign key constraint.

As far as I know, If we don't define `ON DELETE CASCADE` rule in the foreign key constraint, we should manually delete all related records.